examples/IID1-simple-mikrobus-addon-module.mnfs

```diff
@@ -0,0 +1,79 @@
+;
+; mikroBUS Add-on Board Manifest
+; for enc28j60 ethernet controller
+;
+; Copyright 2020 BeagleBoard.org Foundation
+;
+
+[manifest-header]
+version-major = 0
+version-minor = 1
+
+[interface-descriptor]
+vendor-string-id = 1
+product-string-id = 2
+
+; Interface vendor string (id can't be 0)
+[string-descriptor 1]
+string = MikroElektronika
+
+; Interface product string (id can't be 0)
+[string-descriptor 2]
+string = ENC28J60 Ethernet Controller
+
+; mikrobus-descriptor describes the setup of
+; mikroBUS GPIOs and the number of devices
+; present in the add-on board. The GPIO positions 
+; are relative to the mikrobus and the following 
+; values can be used to refer to different GPIOs
+; on the mikrobus port.
+; MIKROBUS_GPIO_INT = 0x01
+; MIKROBUS_GPIO_RST = 0x02
+; MIKROBUS_GPIO_PWM = 0x03
+[mikrobus-descriptor]
+num-devices = 1
+int-gpio = 1
+rst-gpio = 2
+pwm-gpio = 0
+
+; gbphy protocol Bundle 0
+[bundle-descriptor 1]
+class = 0xa
+
+; GPIO protocol on CPort 1
+[cport-descriptor 1]
+bundle = 1
+protocol = 0x2
+
+; SPI protocol on CPort 1
+[cport-descriptor 2]
+bundle = 1
+protocol = 0xb
+
+; device-descriptor describes each device
+; present on the add-on board and other details
+; required by the device driver. The device-descriptor
+; has the following fields,
+; driver-string-id : device driver string id
+; protocol : type of bus the device uses , can be 
+; one of the following: 
+; MIKROBUS_PROTOCOL_SPI = 0x01
+; MIKROBUS_PROTOCOL_I2C = 0x02
+; MIKROBUS_PROTOCOL_UART = 0x03
+; mode : SPI Device Mode
+; irq : IRQ GPIO used by the device
+; irq_type : Type of IRQ used by the
+; device, same values as described 
+; in linux/irq.h
+[device-descriptor 1]
+driver-string-id = 3
+protocol = 0x1
```

Comment:
This was already discussed, and was also posted to the LKML, but we should reuse the CPort Protocol enumerations rather than coming up with new enumerations here for compatibility reasons.